servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/AbstractTestRest.java

```diff
@@ -15,99 +15,22 @@
-import java.util.stream.IntStream;
-import java.util.stream.Stream;
 import javax.annotation.Nullable;
-import org.assertj.core.api.Assumptions;
-import org.assertj.core.groups.Tuple;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.function.Executable;
-import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.CsvSource;
-import org.junit.jupiter.params.provider.MethodSource;
-import org.junit.jupiter.params.provider.ValueSource;
-import org.projectnessie.api.params.FetchOption;
-import org.projectnessie.client.StreamingUtil;
-import org.projectnessie.client.api.CommitMultipleOperationsBuilder;
 import org.projectnessie.client.api.NessieApiV1;
 import org.projectnessie.client.http.HttpClient;
 import org.projectnessie.client.http.HttpClientBuilder;
-import org.projectnessie.client.http.HttpClientException;
-import org.projectnessie.client.rest.NessieBadRequestException;
 import org.projectnessie.client.rest.NessieHttpResponseFilter;
-import org.projectnessie.error.BaseNessieClientServerException;
-import org.projectnessie.error.NessieConflictException;
-import org.projectnessie.error.NessieNotFoundException;
-import org.projectnessie.error.NessieRefLogNotFoundException;
-import org.projectnessie.error.NessieReferenceAlreadyExistsException;
-import org.projectnessie.error.NessieReferenceNotFoundException;
 import org.projectnessie.model.Branch;
-import org.projectnessie.model.CommitMeta;
-import org.projectnessie.model.Content;
-import org.projectnessie.model.Content.Type;
-import org.projectnessie.model.ContentKey;
-import org.projectnessie.model.EntriesResponse;
-import org.projectnessie.model.EntriesResponse.Entry;
-import org.projectnessie.model.IcebergTable;
-import org.projectnessie.model.ImmutableDeltaLakeTable;
-import org.projectnessie.model.ImmutableSqlView;
-import org.projectnessie.model.LogResponse;
-import org.projectnessie.model.LogResponse.LogEntry;
-import org.projectnessie.model.Operation;
-import org.projectnessie.model.Operation.Delete;
-import org.projectnessie.model.Operation.Put;
-import org.projectnessie.model.Operation.Unchanged;
-import org.projectnessie.model.RefLogResponse;
 import org.projectnessie.model.Reference;
-import org.projectnessie.model.ReferenceMetadata;
-import org.projectnessie.model.ReferencesResponse;
-import org.projectnessie.model.SqlView;
-import org.projectnessie.model.SqlView.Dialect;
 import org.projectnessie.model.Tag;
-import org.projectnessie.model.Validation;
 
-public abstract class AbstractTestRest {
-  public static final String COMMA_VALID_HASH_1 =
-      ",1234567890123456789012345678901234567890123456789012345678901234";
-  public static final String COMMA_VALID_HASH_2 = ",1234567890123456789012345678901234567890";
-  public static final String COMMA_VALID_HASH_3 = ",1234567890123456";
+public abstract class AbstractTestRest extends AbstractRestRefLog {
```

Comment:
nit: could you add javadoc to this class about the inheritance model and suggestions about when to add new test superclasses?

Reply:
done